generatebundlefile/data/staging_artifact_move.yaml

```diff
@@ -88,10 +88,10 @@ packages:
         registry: public.ecr.aws/l0g8r8j6
         versions:
           - name: 0.7.1-eks-1-26-38-fe14b98f98132123f478eaf6e65bba4a29bf488b
```

Comment:
Did you miss the suffix here?

Reply:
no it's only for images from 1.27 and [above](https://tiny.amazon.com/1du29girw/IsenLink). The promotion logic is weird and depends on the time it runs and which image is already having the `latest` tag at that time. It certainly needs to be revisited but to your question 1-26 doesn't have that tag on our public ECR.